### DISCUSSION TITLE
Can't find the original Tomb Raider files on Steam

### DISCUSSION THREAD

**Discussion by BENKUN9** [LostArtefacts/TRX#1035] [Q&A]
I'm trying to use the installer to install the files, but I can't seem to get past step 1. The installer can't seem to find my files even when I directly point it to the Tomb Raider folder. Any ideas on this?

**Comment by rr-** [comment]
Hi, the source directory needs to contain a file named `GAME.GOG` (this isn't a mistake – it's used by Steam version as well). LMK if you have this file in your directory, and if pointing the installer to use it changes anything.
